lib/components/MapPolyline.js

```diff
@@ -137,6 +137,8 @@ const propTypes = {
 const defaultProps = {
   strokeColor: '#000',
   strokeWidth: 1,
+  lineJoin: 'round',
```

Comment:
In the docs there is no default for `lineJoin` - could you also update the docs?
tnnx

Reply:
already did, I'm just committing my PR's through the web interface one at a time, sorry about that :)

Reply:
Missed the #1899 PR, could you please make one PR only?  🤗